lib/elixir/lib/calendar/datetime.ex

```diff
@@ -214,10 +214,28 @@ defmodule DateTime do
       iex> datetime
       #DateTime<2018-07-28 12:30:00+02:00 CEST Europe/Copenhagen>
 
+  This function accepts any map or struct that contains at least the same fields as a `NaiveDateTime`
+  struct. The most common example of that is a `DateTime`. In this case the information about the tim
+  zone of that `DateTime` is completely ignored. This is the same principle as passing a `DateTime` to
+  `Date.to_iso8601/2`. `Date.to_iso8601/2` extracts only the date-specific fields (i.e. calendar, year,
+  month and day) of the given structure and ignores all others.
```

Comment:
@lau I have rewritten the first sentence and added more information about the date example. Please let me know what you think!

Reply:
I like it 👍